### PR TITLE
Correct previous proof tests

### PR DESCRIPTION
**Pull request by aljones15**
MINOR release with static fixtures feature and fix for previousProof tests.

### CODE REVIEW

File: inputs/eddsa-rdfc-2022/1.1-previousProofArrayMissingFail-SimpleSigned2.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  ],
+  "id": "urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33",
+  "type": [
+    "VerifiableCredential",
+    "AlumniCredential"
+  ],
+  "name": "Alumni Credential",
+  "description": "A minimum viable example of a VC 1.1 Alumni Credential.",
+  "issuer": "https://vc.example/issuers/5678",
```
Comment (aljones15) [inline inputs/eddsa-rdfc-2022/1.1-previousProofArrayMissingFail-SimpleSigned2.json:21]: @aljones15 issuer might be a did:key so this needs revision.

Comment (aljones15) [inline inputs/eddsa-rdfc-2022/1.1-previousProofArrayMissingFail-SimpleSigned2.json:21]:
```suggestion
  "issuer": "did:key:z6MkhWqdDBPojHA7cprTGTt5yHv5yUi1B8cnXn8ReLumkw6E",
```

File: inputs/eddsa-rdfc-2022/1.1-previousProofArrayOk-SimpleSigned2.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  ],
+  "id": "urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33",
+  "type": [
+    "VerifiableCredential",
+    "AlumniCredential"
+  ],
+  "name": "Alumni Credential",
+  "description": "A minimum viable example of a VC 1.1 Alumni Credential.",
+  "issuer": "https://vc.example/issuers/5678",
```
Comment (aljones15) [inline inputs/eddsa-rdfc-2022/1.1-previousProofArrayOk-SimpleSigned2.json:21]:
```suggestion
  "issuer": "did:key:z6MkhWqdDBPojHA7cprTGTt5yHv5yUi1B8cnXn8ReLumkw6E",
```

File: inputs/eddsa-rdfc-2022/1.1-previousProofMissingFail-SimpleSigned2.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  ],
+  "id": "urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33",
+  "type": [
+    "VerifiableCredential",
+    "AlumniCredential"
+  ],
+  "name": "Alumni Credential",
+  "description": "A minimum viable example of a VC 1.1 Alumni Credential.",
+  "issuer": "https://vc.example/issuers/5678",
```
Comment (aljones15) [inline inputs/eddsa-rdfc-2022/1.1-previousProofMissingFail-SimpleSigned2.json:20]:
```suggestion
  "issuer": "did:key:z6MkhWqdDBPojHA7cprTGTt5yHv5yUi1B8cnXn8ReLumkw6E",
```

File: inputs/eddsa-rdfc-2022/1.1-previousProofArrayMissingFail-SimpleSigned2.json (hunk 1 above)
Comment (aljones15) [reply inputs/eddsa-rdfc-2022/1.1-previousProofArrayMissingFail-SimpleSigned2.json:21]: this was addressed here: https://github.com/Wind4Greg/EdDSA-Test-Vectors/pull/2

**Review by PatStLouis — APPROVED** [review]

**Pull request merged**